appservice/src/intTest/groovy/nu/yona/server/AddDeviceTest.groovy

```diff
@@ -2,108 +2,113 @@ package nu.yona.server
 
 
 import groovy.json.*
-import spock.lang.Shared
 
-class AddDeviceTest extends AbstractAppServiceIntegrationTest {
-
-	@Shared
-	def richardQuin
-
-	def 'Add user Richard'(){
+class AddDeviceTest extends AbstractAppServiceIntegrationTest
+{
+	def 'Set new device request'()
```

Comment:
The test is to verify whether "set" works (and that is in the ``when``). To verify that, "get" is done (as part of ``then``), but that's not what the test is made for, so I prefer to stick with the current name.